feature-libs/storefinder/schematics/add-storefinder/schema.json

```diff
@@ -13,7 +13,7 @@
     },
     "lazy": {
       "type": "boolean",
-      "description": "Lazy load the misc features.",
+      "description": "Lazy load the storefinder features.",
       "default": true
     },
     "features": {
```

Comment:
We don't need the interactive prompt now.

Reply:
Agreed